var/spack/repos/builtin/packages/py-libsonata/package.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyLibsonata(PythonPackage):
+    """SONATA files reader"""
+
+    homepage = "https://github.com/BlueBrain/libsonata"
+    git = "https://github.com/BlueBrain/libsonata.git"
+    pypi = "libsonata/libsonata-0.1.14.tar.gz"
+
+    submodules = True
+
+    maintainers("tristan0x")
+
+    version("develop", branch="master", get_full_repo=True)
+    version("0.1.25", sha256="b332efa718123ee265263e1583a5998eaa945a13b8a22903873764cf1d8173fa")
+
+    depends_on("cmake@3.3:", type="build")
```

Comment:
`CMakeLists.txt` at the source root says the minimum required version is 3.16.  

(Note: That seems to help me get a bit further but I'm still not able to successfully install the package.)

Reply:
I updated the CMake minimum version in b6449ce9480b4b4a263a6f16f3ddc2173333a907
The patch also removes the uses of git submodules in favor of spack packages to fulfill the dependencies.

Reply:
In principle, the names should match so someone familiar with the package and or goes to the repository intuitively know which branch they're using.